core/client/fs/src/main/java/alluxio/client/file/cache/QuotaMetaStore.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.file.cache;
+
+import alluxio.client.quota.CacheScope;
+import alluxio.conf.AlluxioConfiguration;
+import alluxio.exception.PageNotFoundException;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.function.Supplier;
+
+import javax.annotation.Nullable;
+
+/**
+ * A metastore implementation that tracking usage associated with each cache scope.
+ */
+public class QuotaMetaStore extends DefaultMetaStore {
+  /** Track the number of bytes on each scope. */
+  private final Map<CacheScope, Long> mBytesInScope = new ConcurrentHashMap<>();
+  private final Map<CacheScope, CacheEvictor> mCacheEvictors;
```

Comment:
Should we be consistent in the initialization of Maps?

Reply:
fixed